app/products/calls/screens/call_screen/call_screen.tsx

```diff
@@ -373,6 +415,14 @@ const CallScreen = ({componentId, currentCall, participantsDict, teammateNameDis
         return () => listener.remove();
     }, []);
 
+    const goToSettings = useCallback(() => {
+        Permissions.openSettings();
+    }, []);
+
+    const dismissPermissionsError = useCallback(() => {
+        setMicPermissionsErrorDismissed();
+    }, []);
```

Comment:
are this `useCallback`'s needed? can you just do

`onPress={Permissions.openSettings}` and `onPress={setMicPermissionsErrorDismissed}`

Reply:
yep, good catch, thanks!